### PR TITLE
Fix truncation of long text in the Interactive List for IE 11

### PR DESCRIPTION
**Pull request by IKolosynskyi**
### Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### Description

image

### CODE REVIEW

**Comment by IKolosynskyi** [comment]
Fix for #169

File: webpack.config.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /node_modules\/((ansi-styles|strip-ansi|ansi-regex|react-dev-utils|chalk|typescript-plugin-inner-jsx)\/).*/,
+        test: /node_modules\\((ansi-styles|strip-ansi|ansi-regex|react-dev-utils|chalk|typescript-plugin-inner-jsx)\\).*/,
```
Comment (dimabory) [inline webpack.config.js]: Why did you change a regex? It's incorrect now.

Comment (IKolosynskyi) [reply webpack.config.js]: No, it's correct, now. https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats
